addons/crm_livechat/models/discuss_channel.py

```diff
@@ -38,10 +38,13 @@ def execute_command_lead(self, **kwargs):
                 i_start=Markup("<i>"),
                 i_end=Markup("</i>"),
             )
+            self.env.user._bus_send_transient_message(self, msg)
         else:
             lead = self._convert_visitor_to_lead(self.env.user.partner_id, key)
-            msg = _("Created a new lead: %s", lead._get_html_link())
-        self.env.user._bus_send_transient_message(self, msg)
+            msg = Markup(
+                '<div class="o_mail_notification" data-oe-type="create-lead">%s</div>',
+            ) % self.env._("Created a new lead: %s", lead._get_html_link())
```

Comment:
```suggestion
            ) % self.env._("created a new lead: %s", lead._get_html_link())
```
:pick: Looks odd since the author name is added before the notification.
<img width="236" height="41" alt="image" src="https://github.com/user-attachments/assets/9489cad5-86ee-462e-ac4c-f2b328fe48d5" />